hugo/content/docs/langium-overview.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Langium Overview"
+weight: 50
+---
+
+In this chapter you'll get a closer look at the tasks that Langium aims to simplify when creating a custom domain specific language.
+As a fully featured language engineering framework, Langium supports you from parsing a string into a semantic representation of your model up until your interpreter or code generator.
```

Comment:
Another thought... interpreter and code generator are language processors. Also transpilers and maybe meta-programming solutions (ok they are both code generators, but maybe you like a summarizing term) :)